src/General/MB.java

```diff
@@ -79,4 +83,20 @@ public static void show(MBPanel panel) {
         panel.afterVisible();
     }
 
+    /**
+     * Load a image file from the rsc folder via directory/name.
+     *
+     * @param relativePath
+     * @return
+     */
+    public static BufferedImage load(String relativePath){
+        System.out.println(System.getProperty("user.dir"));
+        try {
+            return ImageIO.read(new File(System.getProperty("user.dir") +
+                    File.separator + "rsc" + File.separator + relativePath));
+        } catch (IOException e) {
+            return null;
+        }
+    }
```

Comment:
The following approach for loading images will not work with an executable JAR since everything is packed into one file. The resource folder also should be in the src directory, because it will automatically be packed into the JAR and there will be no problem with the src directory being dissolved.
-> Move rsc directory into the src directory and use the ClassLoader for loading the resources